manifests/image-references

```diff
@@ -1,5 +1,8 @@
 kind: ImageStream
 apiVersion: image.openshift.io/v1
+metadata:
+  annotations:
+    capability.openshift.io/name: OperatorLifecycleManager
```

Comment:
`image-references` doesn't need this annotation; it's consumed by `oc adm release new ...`, which is not aware of `annotations`, just to perform placeholder image replacement.

Reply:
Oh that's good to know, updated the resource, should be good now